### PR TITLE
Update projects-2.13.md

### PR DESCRIPTION
**Pull request by larsrh**

### CODE REVIEW

File: projects-2.13.md [hunk 1]
```diff
@@ -67,15 +69,15 @@ Add in sbt using `libraryDependencies += ...`:
     "com.lihaoyi"                            %% "upack"                   % "0.7.5"
     "com.lihaoyi"                            %% "upickle"                 % "0.7.5"
     "io.argonaut"                            %% "argonaut"                % "6.2.3"
-    "org.foundweekends"                      %% "knockoff"                % "0.8.12"
+    "org.foundweekends"                      %% "knockoff"                 % "0.8.12"
```
Comment (dwijnand) [inline projects-2.13.md]:
```suggestion
    "org.foundweekends"                      %% "knockoff"                % "0.8.12"
```

File: projects-2.13.md [hunk 2]
```diff
@@ -67,15 +69,15 @@ Add in sbt using `libraryDependencies += ...`:
...
     "org.json4s"                             %% "json4s-native"           % "3.6.6"
     "org.rogach"                             %% "scallop"                 % "3.3.0"
     "org.scala-js"                           %% "scalajs-stubs"           % "1.0.0"
     "org.scala-stm"                          %% "scala-stm"               % "0.9.1"
     "org.scalaz"                             %% "scalaz-core"             % "7.2.27"
     "org.squeryl"                            %% "squeryl"                 % "0.9.14"
     "org.typelevel"                          %% "machinist"               % "0.6.8"
-    "ws.unfiltered"                          %% "unfiltered-filter"       % "0.10.0-M4"
+    "ws.unfiltered"                           %% "unfiltered-filter"         % "0.10.0-M4"
```
Comment (dwijnand) [inline projects-2.13.md]:
```suggestion
    "ws.unfiltered"                          %% "unfiltered-filter"       % "0.10.0-M4"
```

**Review by dwijnand — COMMENTED** [review]
Shall we wait for these to resolve?

* https://repo1.maven.org/maven2/org/typelevel/claimant_2.13/
* https://repo1.maven.org/maven2/com/github/mpilquist/simulacrum_2.13/

**Comment by SethTisue** [comment]
> Shall we wait for these to resolve?

Yes, let's not merge until we've verified the artifacts have actually reached Maven Central.

File: projects-2.13.md (hunk 1 above)
Comment (larsrh) [reply projects-2.13.md]: I purposely aligned those again.

Comment (dwijnand) [reply projects-2.13.md]: It's looking aligned in master to me: https://github.com/scala/make-release-notes/blob/2.13.x/projects-2.13.md

and not in your branch: https://github.com/larsrh/make-release-notes/blob/f8f9050ed5e6c14029272e10c9d84a86b117fe12/projects-2.13.md

Comment (larsrh) [reply projects-2.13.md]: Oh. I have ligatures enabled. I take everything back :joy:

**Comment by dwijnand** [comment]
Good to go

**Comment by dwijnand** [comment]
Ah, but merge conflicts

**Pull request merged**